group.go

```diff
@@ -420,6 +431,7 @@ func (cmd *groupCmd) parseFlags(as []string) groupArgs {
 	flags.StringVar(&args.brokers, "brokers", "", "Comma separated list of brokers. Port defaults to 9092 when omitted (defaults to localhost:9092).")
 	flags.StringVar(&args.group, "group", "", "Consumer group name.")
 	flags.StringVar(&args.filter, "filter", "", "Regex to filter groups.")
+	flags.StringVar(&args.tfilter, "tfilter", "", "Regex to filter topics.")
```

Comment:
@michaelritsema i'd prefer a more explicit flag & var name. e.g. `filter-topic` and `filter-group` - what do you think?